core/classes/Misc/ErrorHandler.php

```diff
@@ -39,15 +39,15 @@ public static function catchError(int $error_number, string $error_string, strin
                 break;
 
             case E_USER_WARNING:
-                self::logError('warning', '[' . date('Y-m-d, H:i:s') . '] ' . $error_file . '(' . $error_line . ') ' . $error_number . ': ' . $error_string);
+                self::logError('warning', '[' . date(DATE_FORMAT) . '] ' . $error_file . '(' . $error_line . ') ' . $error_number . ': ' . $error_string);
```

Comment:
This should not be modified as it is not user facing

Reply:
maybe log files should use `DateTimeInterface::ISO8601`